bin/run.js

```diff
@@ -228,39 +226,45 @@ if (coverageReport && !global.__coverage__ && files.length === 0) {
   var args = [nycBin, 'report', '--reporter', coverageReport]
   var child
 
-  // automatically hook into coveralls
+  // automatically hook into coveralls and/or codecov
   if (coverageReport === 'text-lcov' && pipeToService) {
     child = spawn(node, args)
-    var covBin, covName
-
-    if (process.env.COVERALLS_REPO_TOKEN) {
```

Comment:
the main issue of #181 that #183 is fixing. Here we say if else if, in #183 we say forEach